src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -592,13 +529,9 @@ public AttachToCaseEventData(
         return merged;
     }
 
-    private ServiceConfigItem getServiceConfig(CaseDetails caseDetails) {
-        return hasServiceNameInCaseTypeId(caseDetails).flatMap(service -> Try
-            .of(() -> serviceConfigProvider.getConfig(service))
-            .toValidation()
-            .mapError(Throwable::getMessage)
-        )
+    private ServiceConfigItem getServiceConfig(String service) {
```

Comment:
can we rename this method? this is also verifying the update URL

Reply:
`getValidServiceConfig` sounds good enough?

Reply:
let's leave this change for future PRs because the case update related code could be moved from this class